added all the other components

mohd-khan09 opened the pull request:
[1.navbar]
2.footer
3.search bar
4.image grid
![home](https://github.com/sohamtamboli/ampt-icon-generator/assets/126798079/33585388-0352-4038-91c8-97bfef9a9bc5)
![home 2](https://github.com/sohamtamboli/ampt-icon-generator/assets/126798079/b21a65b4-e463-4cfa-9d92-54a08579d7f0)
![profile](https://github.com/sohamtamboli/ampt-icon-generator/assets/126798079/daedcd30-31a3-45df-873a-e308a537542b)
![signin](https://github.com/sohamtamboli/ampt-icon-generator/assets/126798079/18b454a4-afdf-4f7e-b0ef-e30758718889)
![signup](https://github.com/sohamtamboli/ampt-icon-generator/assets/126798079/4c8a89e5-6193-42db-aa3b-ee15a4a6d387)

![Screenshot from 2023-10-26 19-02-19](https://github.com/sohamtamboli/ampt-icon-generator/assets/126798079/693b8968-843d-4e2e-b735-2f5e176f8374)

mohd-khan09 committed:
added all the other components
1.navbar
2.footer
3.search bar
4.image grid

sohamtamboli commented:
(Ampt CI) Starting preview build for `ampt-icon-generator`: `ampt-icon-generator-components(navbar,footer,searchbar,imagegrid)-pr-3`

mohd-khan09 committed:
favourites page added

sohamtamboli commented:
(Ampt CI) Starting preview build for `ampt-icon-generator`: `ampt-icon-generator-components(navbar,footer,searchbar,imagegrid)-pr-3`

mohd-khan09 committed:
1.footer corrected
2.gird 3*3 corrected
3.removed search icon
4.searchbar name changed to promtbar

sohamtamboli commented:
(Ampt CI) Starting preview build for `ampt-icon-generator`: `ampt-icon-generator-components(navbar,footer,searchbar,imagegrid)-pr-3`

mohd-khan09 committed:
dark mode removed from promtbar

mohd-khan09 committed:
check

sohamtamboli commented:
(Ampt CI) Starting preview build for `ampt-icon-generator`: `ampt-icon-generator-components(navbar,footer,searchbar,imagegrid)-pr-3`

sohamtamboli commented:
(Ampt CI) Starting preview build for `ampt-icon-generator`: `ampt-icon-generator-components(navbar,footer,searchbar,imagegrid)-pr-3`

mohd-khan09 committed:
redirect to home page from favourites page

sohamtamboli commented:
(Ampt CI) Starting preview build for `ampt-image-generator`: `ampt-icon-generator-components(navbar,footer,searchbar,imagegrid)-pr-3`

sohamtamboli reviewed (approved):
LGTM
